doc/en/property.md

```diff
@@ -1,56 +1,55 @@
-# 属性
+# Property
 
-有时候我们想基于 better-scroll 做一些扩展，需要对 better-scroll 的一些属性有所了解，下面介绍几个常用属性。
+When you want to extend on better-scroll, there are several properties you may need to understand.
 
 ## x
-  - 类型：Number
-  - 作用：scroll 横轴坐标。
-  
+  - Type: Number.
+  - Usage: scroll horizontal axis coordinate.
+
 ## y
-  - 类型：Number
-  - 作用：scroll 纵轴坐标。
-  
+  - Type: Number.
+  - Usage: scroll vertical axis coordinate.
+
 ## maxScrollX
-  - 类型：Number
-  - 作用：scroll 最大横向滚动位置。
-  - 备注：scroll 横向滚动的位置区间是 0 - maxScrollX，并且 maxScrollX 是负值。
-  
+  - Type: Number.
+  - Usage: max scrollable horizontal coordinate.
+  - Remark: horizontal scroll range is [0, maxScrollX], and maxScrollX is negative value.
+
 ## maxScrollY
-  - 类型：Number
-  - 作用：scroll 最大纵向滚动位置。
-  - 备注：scroll 纵向滚动的位置区间是 0 - maxScrollY，并且 maxScrollY 是负值。
-  
+  - Type: Number.
+  - Usage: max scrollable vertical coordinate could scroll.
+  - Remark: vertical scroll range is [0, maxScrollY], and maxScrollY is negative value.
+
 ## movingDirectionX
-  - 类型：Number
-  - 作用：判断 scroll 滑动过程中的方向（左右）。
-  - 备注：-1 表示从左向右滑，1 表示从右向左滑，0 表示没有滑动。
-  
+  - Type: Number.
+  - Usage: estimate the moving direction on horizontal is left or right.
+  - Remark: -1 means moving from left to right, 1 means moving from right to left, 0 means haven't move.
+
 ## movingDirectionY
-  - 类型：Number
-  - 作用：判断 scroll 滑动过程中的方向（上下）。
-  - 备注：-1 表示从上往下滑，1 表示从下往上滑，0 表示没有滑动。  
-  
+  - Type: Number.
+  - Usage: estimate the moving direction on vertical is up or down during scrolling.
+  - Remark: -1 means down, 1 means down, 0 means haven't move.
+
```

Comment:
-1 means moving from up to down, 1 means moving from down to up, 0 means haven't moved. the same below